packages/image_picker/lib/image_picker.dart

```diff
@@ -8,7 +8,10 @@ import 'dart:io';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart';
 
+/// Denotes that an image is being picked.
 const String kTypeImage = 'image';
+
+/// Denotes that a video is being picked.
```

Comment:
nit: Didn't read the code carefully but I think these can be private. I am not sure if we should hide it at this point as technically it would be a breaking change.

Reply:
I'll just leave it public for now, there's not really a reason to risk hiding it after all this time.